prowler/providers/aws/services/codebuild/codebuild_project_s3_logs_encrypted/codebuild_project_s3_logs_encrypted.metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+  "Provider": "aws",
+  "CheckID": "codebuild_project_s3_logs_encrypted",
+  "CheckTitle": "CodeBuild Project S3 Logs Encrypted",
+  "CheckType": [
+    "Effects/Data Exposure"
+  ],
+  "ServiceName": "codebuild",
+  "SubServiceName": "",
+  "ResourceIdTemplate": "arn:partition:service:region:account-id:resource-id",
+  "Severity": "low",
+  "ResourceType": "AwsCodeBuildProject",
+  "Description": "Encryption of data at rest is a recommended best practice to add a layer of access management around your data. Encrypting the logs at rest reduces the risk that a user not authenticated by AWS will access the data stored on disk. It adds another set of access controls to limit the ability of unauthorized users to access the data.",
+  "Risk": "If the logs are not encrypted, sensitive information could be exposed to unauthorized users.",
```

Comment:
The Description is too long, I think most of that information goes in to the Risk attribute.